egs2/dns_ins20/enh1/local/dns_create_mixture.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+configure=   # Path to the configure file
+
+. utils/parse_options.sh
+. path.sh
+. cmd.sh
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 --configure <configure> <dns> <dns_wav> "
+  echo " where <dns> is dns directory,"
+  echo " <dns_wav> is wav generation space."
+  exit 1;
+fi
+
+dns=$1
+dns_wav=$2
+
+rm -r data/ 2>/dev/null || true
+mkdir -p data/
+
+mod_conf=true
+if [ -z "$configure" ]; then
+  # modify path in the original noisyspeech_synthesizer.cfg
+  configure=${dns}/noisyspeech_synthesizer.cfg
+  train_cfg=data/noisyspeech_synthesizer.cfg
+
+
+  if [ ! -f ${configure} ]; then
+    echo -e "Please check configurtion ${configure} exist"
+    exit 1;
+  fi
+
+  #input datas
+  noise_dir=${dns}/datasets/noise
+  speech_dir=${dns}/datasets/clean
+  noise_dir=/data/DNS-challenge/DNS//datasets/noise
+  speech_dir=/data/DNS-challenge/DNS//datasets/clean
+
+  #outputs
+  noisy_wav=${dns_wav}/noisy
+  clean_wav=${dns_wav}/clean
+  noise_wav=${dns_wav}/noise
+  log_dir=data/log
+
+  #modify the input paths for "\" separated paths
+  sed -e "/^noisy_destination/s#.*#noisy_destination:${noisy_wav}#g" ${configure} | \
```

Comment:
You can actually use a single `sed` command with multiple `-e '...'` arguments to achieve the same goal.

Reply:
Got it!